Domains/0-Core/BisCore.ecschema.xml

```diff
@@ -2333,6 +2338,11 @@
         <ECProperty propertyName="Data" typeName="string" extendedTypeName="Json" description="Encoded style properties" />
     </ECEntityClass>
 
+    <ECEnumeration typeName="TextureFormat" backingTypeName="int" isStrict="true" description="Supported Formats for texture data.">

```

Comment:
We might have to set isStrict to false because we are updating a property that was previously just an int to be an enum.

Reply:
Good catch. Thanks.